packages/fcl/src/transaction/index.js

```diff
@@ -97,10 +114,31 @@ export function transaction(transactionId) {
   }
 
   return {
+    /**
+     * @description Returns the current state of the transaction
+     * @returns {Promise<Transaction>} - Promise that resolves with the transaction
+     */
     snapshot,
+    /**
+     * @description Subscribes to the transaction and provides updates
+     * @param {Function} cb - Callback function that receives updates
+     * @returns {SubscriptionCallback} - Unsubscribe function
```

Comment:
Is `SubscriptionCallback` defined elsewhere / does it need to be?

Reply:
Different subscription callbacks return different types so I've left them separate for now